src/Standards/Generic/Tests/NamingConventions/AbstractClassNamePrefixUnitTest.1.inc

```diff
@@ -56,4 +56,10 @@ class NameAbstractBar {}
 abstract class abstractOkName
 {
 
-}
\ No newline at end of file
+}
+
+final class FinalClassShouldNotTriggerWarning {}
+
+class ChildClassShouldNotTriggerWarning extends AbstractClassName {}
+
+class ClassImplementingInterfaceShouldNotTriggerWarning implements InterfaceName {}
```

Comment:
These tests aren't testing anything which isn't tested already. The sniff does not look at whether the class extends/implements.

Reply:
Yes, I added those tests as an extra precaution, but now I checked the `File::getClassProperties()` code, and I can see that it only looks for tokens before `T_CLASS`, so I agree that they are not necessary. I just removed them.

Reply:
As an alternative, you could have adjusted one or two of the pre-existing tests to have `extends` or `implements`. That would have been fine. But having dedicated tests for it, is redundant.

Reply:
Actually, let me just do that now anyway as I added the extra commit(s) now anyway.